src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -230,114 +223,109 @@ public String toString() {
 	static {
 		loaderThread = new AsyncLoaderThread();
 		loaderThread.start();
 	}
-	
-	private static class AsyncLoaderThread extends Thread {
-		
-		public AsyncLoaderThread() { }
 
+	private static class AsyncLoaderThread extends Thread {
 		@Override
 		public void run() {
 			while (true) {
 				try {
 					loadQueue.take().run();
 				} catch (InterruptedException e) {
+					//noinspection ThrowableNotThrown
 					Skript.exception(e); // Bubble it up with instructions on how to report it
 				}
 			}
 		}
 	}
-	
+
 	static void loadScripts() {
 		final File scriptsFolder = new File(Skript.getInstance().getDataFolder(), Skript.SCRIPTSFOLDER + File.separator);
 		if (!scriptsFolder.isDirectory())
 			scriptsFolder.mkdirs();
-		
+
 		final Date start = new Date();
-		
+
 		Runnable task = () -> {
 			final Set<File> oldLoadedFiles = new HashSet<>(loadedFiles);
-			
+
 			final ScriptInfo i;
-			
+
 			final ErrorDescLogHandler h = SkriptLogger.startLogHandler(new ErrorDescLogHandler(null, null, m_no_errors.toString()));
 			try {
 				Language.setUseLocal(false);
-				
+
 				List<Config> configs = loadStructures(scriptsFolder);
 				i = loadScripts(configs);
 			} finally {
 				Language.setUseLocal(true);
 				h.stop();
 			}
-			
+
 			// Now, make sure that old files that are no longer there are unloaded
 			// Only if this is done using async loading, though!
 			if (loadAsync) {
 				oldLoadedFiles.removeAll(loadedFiles);
 				for (File script : oldLoadedFiles) {
 					assert script != null;
-					
+
 					// Use internal unload method which does not call validateFunctions()
 					unloadScript_(script);
 					Functions.clearFunctions(script);
 				}
 				Functions.validateFunctions(); // Manually validate functions
 			}
-			
+
 			if (i.files == 0)
 				Skript.warning(m_no_scripts.toString());
 			if (Skript.logNormal() && i.files > 0)
 				Skript.info(m_scripts_loaded.toString(i.files, i.triggers, i.commands, start.difference(new Date())));
-			
+
 			SkriptEventHandler.registerBukkitEvents();
 		};
 		if (loadAsync)
 			loadQueue.add(task);
 		else
 			task.run();
 	}
-	
+
 	/**
 	 * Filter for enabled scripts & folders.
 	 */
-	private final static FileFilter scriptFilter = new FileFilter() {
-		@Override
-		public boolean accept(final @Nullable File f) {
-			return f != null && (f.isDirectory() || StringUtils.endsWithIgnoreCase("" + f.getName(), ".sk")) && !f.getName().startsWith("-");
-		}
-	};
-	
+	private final static FileFilter scriptFilter = file ->
+			file != null && (file.isDirectory() || StringUtils.endsWithIgnoreCase("" + file.getName(), ".sk")) && !file.getName().startsWith("-");
+
 	/**
 	 * Loads the specified scripts.
-	 * 
+	 *
 	 * @param configs Configs for scripts, loaded by {@link #loadStructures(File[])}
 	 * @return Info on the loaded scripts.
 	 */
-	public final static ScriptInfo loadScripts(final List<Config> configs) {
+	public static ScriptInfo loadScripts(final List<Config> configs) {

```

Comment:
https://stackoverflow.com/questions/1743715/behaviour-of-final-static-method

Final modifier with static methods has its purpose. I'm not sure if it is important one, though. Opinions?